packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -356,6 +362,46 @@ $table-unsorted-icon-color: get-color-token-from-bg(
   @include table-stacked-header-styles;
 }
 
+@mixin usa-table--sticky-header {
+  border: 1px solid color($theme-table-border-color);
+  border-collapse: separate;
+
+  td,
+  th {
+    border-left: none;
+    border-top: none;
+
+    &:last-child {
+      border-right: none;
+    }
+  }
+
+  tbody tr:last-child {
+    td,
+    th {
+      border-bottom: none;
+    }
+  }
```

Comment:
> **Note**
> To prevent an issue with content showing behind the sticky table headers on scroll, I added `border-collapse: separate`. This caused the cell borders to essentially double up, which created a need to reconfigure how the borders were styled.